include/real_time_tools/mutex.hpp

```diff
@@ -45,7 +52,9 @@ class RealTimeMutex{
                 "error while creating mutex with code %d", res);
     }
   }
-
+  /**
+   * @brief Destroy the RealTimeMutex object
```

Comment:
general question: does it make sense to add this kind of doc which just states the obvious? it clutters the code a lot.

Reply:
We could have it on a single line, it is true.
I am against not at all because it creates a warning during the doxygen build which then makes it hard to see where the documentation is actually needed if we put nothing.